imports/api/methods.js

```diff
@@ -77,18 +77,19 @@ Meteor.methods({
           break;
 
         case "upd-aboutyou":
-          if (Meteor.user.emails.length < 2) {
+          if (Meteor.user().emails.length < 2 && user.emails.address !== undefined) {
             Meteor.users.update(userId, {
               $push: {
                 "emails": user.emails
               }
             }, {
               getAutoValues: false
             });
-          } else if (Meteor.user.emails < 3) {
+          } else if (Meteor.user().emails.length < 3 && user.emails.address !== undefined) {
             Meteor.users.update(userId, {
               $set: {
-                "emails": user.emails
+                "emails.1.address": user.emails.address,
+                "email.1.verified": user.emails.verified
```

Comment:
It comes from here: https://github.com/BancoTalentosTCC/BancoTalentosTCC/blob/master/imports/ui/pages/panel/settings/index.js#L81